Sources/Jobs/QueuedJob.swift

```diff
@@ -21,10 +21,13 @@ public struct QueuedJob<JobID: Sendable>: Sendable {
     public let id: JobID
     /// Job data
     public let jobBuffer: ByteBuffer
+    /// Job queued at
+    public let queuedAt: Date
 
     /// Initialize a queue job
-    public init(id: JobID, jobBuffer: ByteBuffer) {
+    public init(id: JobID, jobBuffer: ByteBuffer, queuedAt: Date = .init()) {
```

Comment:
queuedAt should not have a default value. It means we are missing places where we aren't setting the value correctly

Reply:
I set a default here because, I don't think we have any use for this field in the Redis Driver implementation. I will drop the default then.